dashboard/app/models/user.rb

```diff
@@ -425,7 +425,7 @@ def district_teachers(cohort = nil)
   validates :name, length: {within: 1..70}, allow_blank: true
   validates :name, no_utf8mb4: true
 
-  defer_age = proc {|user| user.provider == 'google_oauth2' || user.provider == 'clever'}
+  defer_age = proc {|user| user.provider == 'google_oauth2' || user.provider == 'clever' || user.provider == User::PROVIDER_SPONSORED}
```

Comment:
Since these students were previous created via pegasus, we didn't do any of this validation, now that I'm doing it via dashboard it fails. Updating age requirements on User model to get this new API working.

Reply:
I think it's pretty time consuming for teachers to have to input themselves instead of having students update it themselves when they first login.